File: integration-tests/gradle/src/main/resources/build-configuration/with-application-properties/src/main/java/org/acme/ExampleResource.java
package org.acme;

import jakarta.inject.Inject;
import jakarta.ws.rs.GET;
import jakarta.ws.rs.Path;
import jakarta.ws.rs.Produces;
import jakarta.ws.rs.core.MediaType;

import org.eclipse.microprofile.config.inject.ConfigProperty;

@Path("/hello")
public class ExampleResource {
    @GET
    @Produces(MediaType.TEXT_PLAIN)
    public String hello() {
        return "hello";
    }

}

File: integration-tests/gradle/src/main/resources/multi-module-kotlin-project/web/src/main/kotlin/com/example/quarkusmm/ExampleResource.kt
package com.example.quarkusmm

import com.example.quarkusmm.port.CustomerService
import jakarta.inject.Inject
import jakarta.ws.rs.GET
import jakarta.ws.rs.Path
import jakarta.ws.rs.Produces
import jakarta.ws.rs.core.MediaType

@Path("/hello")
class ExampleResource {

    @Inject
    lateinit var service: CustomerService

    @GET
    @Produces(MediaType.TEXT_PLAIN)
    fun hello() = service.getMessage()
}